scripts/run_messaging_hub.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+
+import argparse
+# import multiprocessing
+import sys
+import threading
+import time
+
+from pocs.utils.config import load_config
+from pocs.utils.logger import get_root_logger
+from pocs.utils.messaging import PanMessaging
+
+
+def run_forwarder(logger, sub_port, pub_port):
+    msg = 'Starting {} -> {} forwarder'.format(sub_port, pub_port)
+    print(msg)
+    logger.info(msg)
+    try:
+        PanMessaging.create_forwarder(sub_port, pub_port)
+    except Exception:
+        pass
+    msg = 'Forwarder for {} -> {} has stopped'.format(sub_port, pub_port)
```

Comment:
Maybe we can catch a `KeyboardInterrupt` and show an `info` message but on other `Exceptions` show a warning?

Reply:
I see you catch Ctrl-C below so above comment might not be valid.

Reply:
Right, the interrupt is handled on the main thread only IIUC.